openmapflow/notebooks/compare_ceo_labels.ipynb

```diff
@@ -0,0 +1,2563 @@
+{
```

Comment:
<p>What about something like:</p><pre class="ql-syntax" spellcheck="false">ceo_set1_path = # Your first csv label file, e.g., ceo-Hawaii-Jan-Dec-2020-(Set-1)-sample-data-2022-08-16.csv
</pre><p>To make it more clear that this is your own path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>